File: app/src/main/java/com/narcoding/localnotepad/Activity/FullImageActivity.java
package com.narcoding.localnotepad.Activity;

import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.graphics.drawable.BitmapDrawable;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Base64;
import android.widget.ImageView;
import android.widget.RelativeLayout;

import com.narcoding.localnotepad.R;

public class FullImageActivity extends AppCompatActivity {
    private ImageView img_fullScreen;
    private RelativeLayout rl;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_full_image);
        img_fullScreen= (ImageView) findViewById(R.id.img_fullScreen);
        rl= (RelativeLayout) findViewById(R.id.rl_activity_full_image);

        byte[] byteArray = getIntent().getByteArrayExtra("image");
        Bitmap bmp = BitmapFactory.decodeByteArray(byteArray, 0, byteArray.length);
        img_fullScreen.setImageBitmap(bmp);
        //BitmapDrawable bitmapDrawable = new BitmapDrawable(getResources(), bmp);
        //rl.setBackgroundDrawable(bitmapDrawable);

    }

}

File: app/src/main/java/com/narcoding/localnotepad/view/FullImageActivity.kt
package com.narcoding.localnotepad.view

import android.graphics.BitmapFactory
import android.os.Bundle
import android.view.View
import android.widget.ImageView
import android.widget.RelativeLayout
import androidx.appcompat.app.AppCompatActivity
import com.narcoding.localnotepad.R

class FullImageActivity : AppCompatActivity() {
    private var img_fullScreen: ImageView? = null
    private var rl: RelativeLayout? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_full_image)
        img_fullScreen = findViewById<View>(R.id.img_fullScreen) as ImageView
        rl = findViewById<View>(R.id.rl_activity_full_image) as RelativeLayout
        val byteArray = intent.getByteArrayExtra("image")
        val bmp = BitmapFactory.decodeByteArray(byteArray, 0, byteArray!!.size)
        img_fullScreen!!.setImageBitmap(bmp)
        //BitmapDrawable bitmapDrawable = new BitmapDrawable(getResources(), bmp);
        //rl.setBackgroundDrawable(bitmapDrawable);
    }
}